### PR TITLE
PKG: Add convexhull computation package

### PR DESCRIPTION
**Pull request by Shaikh-Ubaid**
towards https://github.com/lcompilers/lpython/issues/992

### CODE REVIEW

**Comment by Shaikh-Ubaid** [comment]
Ready.

File: integration_tests/lpconvexhull/lpconvexhull_main.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    Args:
+        points: A list of points.
+
+    Returns:
+        A list of points that form the convex hull.
+    """
+
+    n: i32 = len(points)
+    if n < 3:
+        return [(-1, -1)]  # Convex hull not possible
```
Comment (Shaikh-Ubaid) [inline integration_tests/lpconvexhull/lpconvexhull_main.py:31]: It seems that we do not support returning empty lists (https://github.com/lcompilers/lpython/issues/1841). Therefore, returning a list with single item with negative/invalid coordinates.

**Review by czgdp1807 — APPROVED** [review]

File: integration_tests/CMakeLists.txt [hunk 2]
```diff
@@ -528,6 +528,7 @@ RUN(NAME test_package_01     LABELS cpython llvm)
 RUN(NAME test_pkg_lpdraw     LABELS cpython llvm wasm)
 RUN(NAME test_pkg_lnn_01        LABELS cpython llvm)
 RUN(NAME test_pkg_lnn_02        LABELS cpython llvm)
+RUN(NAME test_pkg_lpconvexhull  LABELS cpython c)
```
Comment (certik) [inline integration_tests/CMakeLists.txt:531]: Does it work with llvm?

Comment (Shaikh-Ubaid) [reply integration_tests/CMakeLists.txt:531]: >Does it work with llvm?

It does not work with LLVM yet.

**Review by certik — APPROVED** [review]
In that case I think this looks good. Let's merge the sync PR first .

**Pull request merged**